make.xsh

```diff
@@ -133,30 +133,56 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def run_local_db():
     """
     Use the "database-backup" service in the "docker-compose.yml" file to backup
     all of the tables in the instace of Postgres running locally.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+
+@makefile.target(tag="run services locally")
+def backup_local_db():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally. You must have a
+    postgres container running which you can run using "make run-local-db"
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
     docker-compose run database-backup backup
 
 @makefile.target(tag="run services locally")
-def restore_database():
+def restore_local_db():
     """
-    Use the "database-backup" service in the "docker-compose.yml" file to identify
-    the most recent backup in "infrastructure/containers/postgres/backups" and load that data
-    into the database.
+    Use the "database-backup" service in the "docker-compose.yml" file to backup
+    all of the tables in the instace of Postgres running locally.
+    """
+    export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
+    docker-compose run database-backup restore-from-most-recent
 
-    Note that this will wipe the existing data in the database first.
+@makefile.target(tag="run services locally")
+def start_database_stack():
+    """
+    Use the "database-backup" service in the "docker-compose.yml" file to backup on a
+    regular inteval specified in the "docker-compose.yml" all of the tables in the
+    instace of Postgres that is running.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
-    docker network prune --force
-    database_backup_container_id = $(docker ps --quiet --filter name=database-backup)
-    database_backup_container_id = database_backup_container_id.strip()
-    docker exec @(database_backup_container_id) python3 backup_or_restore.py restore-from-most-recent
+    export_rootski_profile_aws_creds()
+    $POSTGRES_HOST = get_localhost()
+    # Runs a postgres instance if one is not already running
+    docker ps | grep postgres || docker-compose run -d -p $POSTGRES_PORT:$POSTGRES_PORT postgres
```

Comment:
I see what you're trying to do here. If we start these with the `docker-compose` command, they won't self heal. Can you pick one a target to keep (either this one or `start-local-database` and just name it `start-database`?

Rather than use two `docker-compose run ...` commands, you can just do `docker stack deploy --compose-file docker-compose.yml rootski-database`. That will start ALL services in the `docker-compose.yml` file. (so both the database and the backup service) and they will both self heal if they go down.